### PR TITLE
Dead code removal

### PR DESCRIPTION
**Pull request by amosr**
Especially for extra buffers produced from latest

### CODE REVIEW

File: test/Icicle/Test/Avalanche/Flatten.hs [hunk 1]
```diff
@@ -38,7 +38,7 @@ namer = AC.namerText (flip Var 0)
 
 
 -- Flattening - removing all folds keeps value same
-prop_flatten_commutes_value t =
+zprop_flatten_commutes_value t =
```
Comment (tranma) [inline test/Icicle/Test/Avalanche/Flatten.hs]: why is this test disabled again?

**Comment by tranma** [comment]
nice. Would be great to have a test with some dead code being removed? :dancers:

File: test/Icicle/Test/Avalanche/Flatten.hs (hunk 1 above)
Comment (amosr) [inline test/Icicle/Test/Avalanche/Flatten.hs]: haha sorry. that was while I was running the tests from ghci

**Comment by amosr** [comment]
yeah.. maybe I should add another test. but one of the buffers does disappear from an avalanche test!

**Comment by jystic** [comment]
:vibration_mode:

**Pull request merged**